queries.sql

```diff
@@ -95,14 +95,14 @@ GROUP BY species;
 
 /* query using JOIN */
 
-/* Join owner_full_name using the owners_id */
+/* What animals belong to Melody Pond? */
 
 SELECT animals.name
```

Comment:
- [ ] Like the previous CR mentioned you should use SELECT animals instead of SELECT animals.name according to the requirements